### PR TITLE
Tests the data segment can be overwritten

### PR DESCRIPTION
**Pull request by hugo-dc**
Currently wagon fails when a deployer contract has two `data` statement writing to the same memory slot, if the data is the same it works ok, but if it is different data then the result is different as in hera.

### CODE REVIEW

**Review by gballet — APPROVED** [review]
That exactly what the problem is, thanks @hugo-dc

File: src/GeneralStateTestsFiller/stEWASMTests/overwriteDataFiller.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          balance: '99999929052'
+        # newly-created contract
+        6295ee1b4f6dd65047762f924ecd367c17eabf8f:
+          code: '0x0061736d01000000010401600000030201000503010001071102066d656d6f72790200046d61696e00000a05010300010b'
+  transaction:
+    data:
+    - |
+      (module
+        (import "ethereum" "finish" (func $finish (param i32 i32)))
+        (memory 100)
```
Comment (axic) [inline src/GeneralStateTestsFiller/stEWASMTests/overwriteDataFiller.yml]: Please don't use memory 100.

Comment (gballet) [reply src/GeneralStateTestsFiller/stEWASMTests/overwriteDataFiller.yml]: @axic why not? The reason I'm asking is that this tx breaks the sync with wagon so I'm wondering if you could pinpoint a problem.

Comment (axic) [reply src/GeneralStateTestsFiller/stEWASMTests/overwriteDataFiller.yml]: We have (should have?) specific tests for high memory usage. Other tests which do not need that should not use it.

Comment (hugo-dc) [reply src/GeneralStateTestsFiller/stEWASMTests/overwriteDataFiller.yml]: @axic fixed

**Review by axic — APPROVED** [review]

**Pull request merged**